src/app/fdctl/topology.h

```diff
@@ -350,11 +353,12 @@ fd_topo_link_kind_str( ulong kind ) {
     case FD_TOPO_LINK_KIND_VERIFY_TO_DEDUP: return "verify_dedup";
     case FD_TOPO_LINK_KIND_DEDUP_TO_PACK:   return "dedup_pack";
     case FD_TOPO_LINK_KIND_GOSSIP_TO_PACK:  return "gossip_pack";
-    case FD_TOPO_LINK_KIND_LSCHED_TO_PACK:  return "lsched_pack";
+    case FD_TOPO_LINK_KIND_STAKES_TO_FD:    return "stakes_fd";
     case FD_TOPO_LINK_KIND_PACK_TO_BANK:    return "pack_bank";
     case FD_TOPO_LINK_KIND_POH_TO_SHRED:    return "poh_shred";
     case FD_TOPO_LINK_KIND_SHRED_TO_NETMUX: return "shred_netmux";
     case FD_TOPO_LINK_KIND_SHRED_TO_STORE:  return "shred_store";
+    case FD_TOPO_LINK_KIND_CONTACT_TO_SHRED:return "contact_shred";
```

Comment:
I don't like that this is 7 chars for "contact", I think monitor caps out at 6 and it's nice to keep short for display. Maybe name "cntact_shred", or rename the whole thing? GOSSIP vs. CONTACT is pretty vague here, since they are both from the same place?